src/AWS.Deploy.Common/Recipes/OptionSettingItem.cs

```diff
@@ -57,6 +57,17 @@ public partial class OptionSettingItem : IOptionSettingItem
         /// </summary>
         public string Id { get; set; }
 
+        /// <summary>
+        /// The Id of the option setting item in the Recipe Option Setting tree.
+        /// This helps easily reference the Option Setting without context of the parent setting.
+        /// </summary>
+        public string LeafId { get; set; }
```

Comment:
`FullyQualifiedId` or something like that would make more sense to me then `LeafId`. When I think of leaf I expected it to be only the last part of the id which is the same as the id field.

Reply:
Done